src/deck-options.md

```diff
@@ -4,260 +4,219 @@
-introduce new cards, you can do so by suspending the reviews, or increasing your
-review limit. That said, it is recommended you hold off on new cards until you
-catch up instead, as introducing more new cards when you're behind will only
-make the backlog worse.
+Anki provides three options for daily limits:
 
-From Anki 2.1.61 this feature is optional, and can be deactivated globally from the
-deck options screen.
+- **Preset**: applies to all the decks using a preset.
+- **This deck**: specific to a particular deck.
+- **Today only**: specific to a particular deck, and temporary.
 
-### Per-Deck Daily Limits
+### New Cards Ignore Review Limit
 
-From version 2.1.55 it is possible to use the same preset for different decks / subdecks, with customized
-limits for each one of them. This eliminates the need to create cloned presets just for that
-purpose, and makes it easier to set custom limits on sub-decks when you have many nested decks.
+By default, the review limit also applies to new cards, and no new cards will be shown when the review limit has been reached. If this option is enabled, new cards will be shown regardless of the review limit. 
 
-The options are:
 
-- Preset: The limit is shared with all decks using this preset.
-- This deck: The limit is specific to this deck.
-- Today only: Make a temporary change to this deck's limit.
+If you have a backlog of overdue review cards, it is recommended that you stop introducing new cards until you
+catch up with that backlog. Continuing to introduce new cards when you're already behind can 
+make the backlog worse.
+
+### Limits Start From The Top
+
+By default, the daily limits of a higher-level deck do not apply if you're studying from its subdeck. However, if this option is enabled, the limits will start from the top-level decks instead, which can be useful if you wish to study individual subdecks, while enforcing a total limit on cards for the deck tree. Enabling this option for any preset will affect all decks and presets in your collection.
 
 ## New Cards
 
-The settings in this section only affect new cards and cards in initial
-[learning](studying.md#learningrelearning-cards) mode. Once a card
-has graduated (i.e. there are no more learning steps for this card), it becomes a
-[review card](studying.md#review-cards), and the
-settings in this section are no longer applicable.
+The options here only affect new cards and
+[learning cards](studying.md#learningrelearning-cards). Once a card
+has graduated (that is, gone through all the learning steps), the options in this section no longer apply to the card.
 
 ### Learning Steps
 
 Controls the number of learning repetitions, and the delay
 between them. One or more delays, separated by spaces must be entered.
-Each time you press `Good` during review, the card moves to the next step.
+Each time you click "Good" during review, the card moves to the next step. Each time you click "Again", the card goes back to the first step.
```

Comment:
I think backticks were better in this section.

Reply:
@dae Because we are talking of buttons, we actually should be using bold. Do I do this in this PR or can follow up later?

Reply:
If you're happy to do a follow-up, that will let us get this one over the line.